lucille-core/src/main/java/com/kmwllc/lucille/stage/ParseDate.java

```diff
@@ -40,20 +37,48 @@ public class ParseDate extends Stage {
   private final List<String> sourceFields;
   private final List<String> destFields;
   private final UpdateMode updateMode;
+  private final String timeZoneId;
+
+  private static final String DEFAULT_TIME_ZONE_ID = Calendar.getInstance().getTimeZone().getID();
```

Comment:
I'd call this "localTimeZoneId" instead of "DEFAULT_TIME_ZONE_ID"
reasons:

1. yes, it's the default for the system where lucille is being run at the moment, but it's not a global, unchanging default -- same lucille distribution could have different values here depending on what server it's executed on
2. yes, it's a private static final String, but Calendar.getInstance().getTimeZone().getID() isn't a "constant expression" as described above, so I think this is a scenario where the uppercase convention shouldn't apply